data/crates.json

```diff
@@ -27,6 +27,17 @@
                             }, {
                                 "name": "chrono",
                                 "notes": "The most comprehensive and full-featured datetime library, but more complex because of it."
+                            }],
+                            "see_also": [{
+                                "name": "jiff",
+                                "link": "https://crates.io/crates/jiff",
+                                "notes": "A relatively new general-purpose datetime library that aims to address some of the pain points with time and chrono."
```

Comment:
FWIW, I used this style of description (described in terms of other projects) with ripgrep when it was released ("it's like grep and ag") and I kinda regret it. Consider that if you're looking at this list for crate recommendations, you probably _don't_ already know about `time` and `chrono`. And so describing Jiff, in brief, with what it isn't, probably doesn't help too much. On the other hand, there really isn't the space here (as is intended) to do even a concise cost/benefit analysis.

It is _very_ hard to do this in one sentence, but here's my crack at it:

> A datetime library with automatic tzdb integration and convenient daylight saving time calendar arithmetic, but currently lacks some of the ecosystem integration that `time` and `chrono` have.